lib/widgets/content.dart

```diff
@@ -653,6 +655,10 @@ class _InlineContentBuilder {
   InlineSpan _buildStrong(StrongNode node) => _buildNodes(node.nodes,
     style: weightVariableTextStyle(_context, wght: 600, wghtIfPlatformRequestsBold: 900));
 
+
+  InlineSpan _buildDeleted(DeletedNode node) => _buildNodes(node.nodes,
```

Comment:
nit: just one blank line to separate:
```suggestion
    style: weightVariableTextStyle(_context, wght: 600, wghtIfPlatformRequestsBold: 900));

  InlineSpan _buildDeleted(DeletedNode node) => _buildNodes(node.nodes,
```